src/robusta/core/sinks/robusta/robusta_sink.py

```
@@ -556,6 +615,16 @@ def __discovery_watchdog(self):
             time.sleep(DISCOVERY_WATCHDOG_CHECK_SEC)
         logging.warning("Watchdog finished")
 
+    def __discover_namespace_resources(self):
+        logging.info("Namespace Resources discovery initialized")
+        while self.__activ:
+            start_t = time.time()
```

Ruff (0.12.2)

Local variable `start_t` is assigned to but never used

Remove assignment to unused variable `start_t`

(F841)